src/scmdata/ops.py

```diff
@@ -1,16 +1,19 @@
 """
 Operations for :obj:`ScmRun`
 
-These rely on
+These largely rely on
 `Pint's Pandas interface <https://pint.readthedocs.io/en/0.13/pint-pandas.html>`_
 to handle unit conversions automatically
 """
+import warnings
+
 import pandas as pd
 import pint_pandas
+import scipy
 from openscm_units import unit_registry
 
 
-def prep_for_op(inp, op_cols, ur=unit_registry):
+def prep_for_op(inp, op_cols, meta, ur=unit_registry):
```

Comment:
What is the purpose of `meta`? It looks like it is always `self.meta.columns` which is the default no?

Reply:
It means we can control the order in which the meta columns are returned. If you don't have it, things might blow up if ``other`` returns a different timeseries index order than ``self``.